spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe PostsController do
+  describe 'GET #index' do
+    let!(:post) { FactoryGirl.create(:post) }
+    before { get :index }
+
+    it { is_expected.to render_template 'index' }
+
+    it 'assigns @posts' do
+      expect(assigns :posts).to eq [post]
+    end
+  end
+
+  describe 'GET #new' do
+    before { get :new }
+
+    it { is_expected.to render_template 'new'}
+
+    it 'assigns a new Post to @post' do
+      expect(assigns :post).to be_a_new(Post)
```

Comment:
Style/NestedParenthesizedCalls: Add parentheses to nested method call `assigns :post`.